pagopa/io-app
# chore: [IOBP-588] Add radio group into PSP sorting bottom sheet

## Short description
This PR adds the radio group into the PSP sorting bottom sheet in order to keep track the order type selected.

## List of changes proposed in this pull request
- Replaced the `ListItem`s with the `RadioGroup` into `useSortPspBottomSheet` hook

## How to test
Proceed with a payment flow with the new payment FF enabled, then inside the PSP selection, you should be able to see a radio group inside of that bottom sheet that keep track of the order type selected

## Preview

https://github.com/pagopa/io-app/assets/34343582/cd6dfc5d-7680-408a-bf38-e7335b938224
